qiime2/core/transform.py

```diff
@@ -63,14 +63,14 @@ def make_transformation(self, other, recorder=None):
                      input_record=self._record, output_name=other._view_name,
                      output_record=other._record)
 
-        def transformation(view):
+        def transformation(view, validate_level='min'):
             view = self.coerce_view(view)
-            self.validate(view)
+            self.validate(view, validate_level)
 
             new_view = transformer(view)
 
             new_view = other.coerce_view(new_view)
-            other.validate(new_view)
+            other.validate(new_view, validate_level)
```

Comment:
Sorry @Oddant1, realized as I was walking in, we probably don't want to validate the result of the transform at `max`. As then we would be scanning sequence data twice on import.